tests/test_develop.py

```diff
@@ -99,3 +102,22 @@ def test_uninstall(site_packages: Path, conda_pth: Path):
         _uninstall(site_packages, path)
 
         assert list(filter(None, conda_pth.read_text().split("\n"))) == develop_paths
+
+
+def test_get_setup_py(conda_pth: Path):
```

Comment:
The `conda_pth` fixture you use here does not appear to be used since it's overwritten on line 109?